static/app/utils/fields/index.ts

```diff
@@ -1438,6 +1440,11 @@ const REPLAY_CLICK_FIELD_DEFINITIONS: Record<ReplayClickFieldKey, FieldDefinitio
     kind: FieldKind.FIELD,
     valueType: FieldValueType.STRING,
   },
+  [ReplayClickFieldKey.CLICK_COMPONENT_NAME]: {
+    desc: t('the name of the component that was clicked'),
```

Comment:
I wonder how we can clarify this a bit more -- i.e. as an user I don't think I would associate this to mean my React component names

I also wonder if it's possible to link to docs from here, it may be a bit awkward since it's a dropdown though.

Reply:
Yeah, I'd like to include more details here but it's hard to fit without cluttering the dropdown. We can certainly revisit this in the near future.

For an immediate change, maybe we could say `the name of the *frontend* component that was clicked` instead?